packages/@atjson/source-ckeditor/src/source.ts

```diff
@@ -25,9 +25,29 @@ function fromNode(
       attributes
     });
   } else if (isElement(node)) {
+    content += "\uFFFC";
```

Comment:
It may be helpful to represent the content as the pseudo-html structure that CKEditor uses internally in its inspector.

cf.

```ts
content += `<${node.name}>`
```